src/components/GlobalNavigation/components/DesktopView/components/CollectionLayout/CollectionLayout.tsx

```diff
@@ -0,0 +1,81 @@
+import React from 'react';
+import { useGlobalNavigationStateContext } from '~/components/GlobalNavigation/GlobalNavigation.context';
+import { PromotionCard } from '~/components/GlobalNavigation/components/PromotionCard';
+import { useMenuItemContext } from '~/components/GlobalNavigation/components/DesktopView/components/MenuItem/MenuItem.context';
+import { CollectionList } from '../CollectionList';
+import { CollectionImage } from '../CollectionImage';
+import { getCollectionLists } from './CollectionLayout.utils';
+import type { CollectionLayoutType } from './CollectionLayout.types';
+import styles from './CollectionLayout.module.css';
+
+const CollectionLayout: CollectionLayoutType = ({
+  articles,
+  articlesListHeading,
+  id,
+  image,
+  items,
+  promotion,
+  topLevelCollectionLabel,
+  type,
+}) => {
+  const { activeCollectionId, isOpen } = useGlobalNavigationStateContext();
+  const { id: currentId } = useMenuItemContext(id);
+
+  const {
+    nestedCollections,
+    notableNestedCollections,
+    taxonomyOfDesignElement,
+    topLevelCollections,
+  } = getCollectionLists(items);
+
+  return (
+    <>
+      <div className={styles.collectionsWrapper}>
+        <CollectionList
+          heading={topLevelCollectionLabel || 'Category'}
```

Comment:
Not sure if it's likely that `topLevelCollectionLabel` would be intentionally set to an empty string but if it happens the outcome could be unexpected.

```suggestion
          heading={topLevelCollectionLabel ?? 'Category'}
```

Reply:
This line needs to be changed all together - need a translated variable for `Category`. thank you for the pick up